src/components/badge/themes/news.json

```diff
@@ -0,0 +1,57 @@
+{
+	"badge": {
+		"styles": {
+			"default": {
+				"align-self": "baseline",
+				"border-radius": "var(--border-radius-pill)",
+				"color": "var(--global-white)",
+				"font-size": "var(--body-font-size-tiny)",
+				"font-weight": "var(--global-font-weight-7)",
+				"line-height": "var(--body-line-height-tiny)",
+				"padding": "var(--global-spacing-1) var(--global-spacing-2)"
+			}
+		}
+	},
+	"badge-danger": {
```

Comment:
I think we either need to put the baseline styles here and only override the individual styles or vice versa... having the same style definitions both here and in the individual components makes it confusing as to which is selected.